pkg/operator/resource/resourceapply/admissionregistration.go

```diff
@@ -0,0 +1,185 @@
+package resourceapply
+
+import (
+	"context"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	admissionregistrationclientv1 "k8s.io/client-go/kubernetes/typed/admissionregistration/v1"
+	"k8s.io/klog"
+	"k8s.io/utils/pointer"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// The Apply<type> methods in this file ensure that a resource is created or updated to match
```

Comment:
this big block of go doc must go to doc.go. This describes generic behaviour that must not differ by resource. Obviously, this poses the question whether we have a generic behaviour. If not, we have to make it uniform.